### PR TITLE
fix(docz-tools): Make Atlantis theme responsivise.

### PR DESCRIPTION
**Pull request by eddysims**
## Motivations

Atlantis is not very usable on a mobile device. This PR will make Atlantis work on a mobile device
 
### Added

- 

### Changed

- Moved Layout component to use CSS modules
- Added responsive styles to Layout and Playground

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/docz-tools/src/gatsby-theme-docz/components/Layout/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    width: sideBarWidth,
+    left: open ? 0 : -sideBarWidth,
+  };
+  const menuStyle = { left: open ? sideBarWidth : 0 };
+
+  return (
+    <>
+      <div className={styles.layout}>
+        <div className={styles.menu} style={menuStyle}>
+          <Button icon={open ? "remove" : "menu"} onClick={handleClick} />
```
Comment (chris-at-jobber) [inline packages/docz-tools/src/gatsby-theme-docz/components/Layout/Layout.tsx:29]: can we make this a `tertiary` button?

**Comment by chris-at-jobber** [comment]
@seanhealy can you take a look? Just the one comment from me

**Review by chris-at-jobber — CHANGES_REQUESTED** [review]
just changing the toggle button type and I'm good

**Comment by chris-at-jobber** [comment]
A question, non-blocking: is it possible to have the menu close when tapping the overlay?

File: packages/docz-tools/src/gatsby-theme-docz/components/Layout/Layout.module.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 8888;
+  width: 100%;
+  height: 100vh;
+  background: var(--color-surface);
```
Comment (chris-at-jobber) [inline packages/docz-tools/src/gatsby-theme-docz/components/Layout/Layout.module.css:28]: Another thought, also non-blocking... we should use a consistent color as with Menu and Modal (greyBlue at 80% opacity).

I was thinking as I did the color swaps in our components that I missed the opportunity to add a `color-overlay` variable, but looking at those components we actually implement in different ways. Menu uses an [rgba value](https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Menu/Menu.css#L111) equivalent to greyBlue at 80%, while Modal uses [greyBlue and sets an opacity](https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Modal/Modal.css#L29) on the element.

**Comment by eddysims** [comment]
@chris-at-jobber I have addressed your comments.

Can you also take a look at `colors.css` and take a look at how I have approached the `--color-overlay`. I think that this is a good way to go with this, but would like your thoughts as well.

File: packages/design/src/colors.css [hunk 3]
```diff
@@ -106,7 +108,8 @@
   --color-grey--lightest: rgb(244, 244, 244);
   --color-grey--dark: rgb(118, 118, 118);
 
-  --color-greyBlue: rgb(101, 120, 132);
+  --color--greyBlue--rgb: 101, 120, 132;
```
Comment (chris-at-jobber) [inline packages/design/src/colors.css:111]: Ooooh this kind of blew my mind, I think this is good! I had to read it a few times before I realized _how_ it was working, but this is, I think, a good solution.

**Review by chris-at-jobber — APPROVED** [review]
Looks good to me! Might be good to have @seanhealy or @hosseintalebi take a look as well but this is a nice fix.

**Pull request merged**